scenario/mocking.py

```diff
@@ -631,6 +632,13 @@ def resource_get(self, resource_name: str) -> str:
                 f"resource {resource_name} not found in State. please pass it.",
             )
 
+    def credential_get(self) -> CloudSpec:
+        if not self._state.cloud_spec:
+            raise ModelError(
+                "ERROR cloud spec is empty, initialise it with `scenario.State(cloud_spec=scenario.state.CloudSpec(...))`",
```

Comment:
@PietroPasotti what's your preference for adding to [`__init__`](scenario/__init__)? Would you generally have all the `State` objects in there? If so I presume this would be:

```suggestion
                "ERROR cloud spec is empty, initialise it with `scenario.State(cloud_spec=scenario.CloudSpec(...))`",
```

And `CloudSpec` would be added to the `__all__` list.

Reply:
yes, it should be a toplevel name